### ISSUE TITLE
This seems to be broken

### ISSUE THREAD

**Issue by guibor** [SavchenkoValeriy/emacs-powerthesaurus#11]
Presumably a change in how powerthesaurus.org returns results

**Comment by maxchaos** [comment]
It would appear that powerthesaurus changed both where it keeps the json store (that this package uses to extract suggestions) and its structure. I tried to make sense out of it but it seemed unnecessarily complex, so I modified the package to extract synonyms from the html body instead.
If @SavchenkoValeriy is ok with it, I can open a pull request.

**Comment by lyjdwh** [comment]
@maxchaos 
it seems @SavchenkoValeriy temporarily has no time to handle this. So you can first commit it to your fork that we can use it :)

**Comment by maxchaos** [comment]
I've already done so in a different branch, namely [pt-api-change](https://github.com/maxchaos/emacs-powerthesaurus/tree/pt-api-change). I believe I should wait for @SavchenkoValeriy's response a bit longer before I merge it into `master` (unless this is inconvenient to you), since he may be able to fix this without dropping json store support.

**Comment by lyjdwh** [comment]
It's ok. I am using your pt-api-change branch now, thanks!

**Comment by d1egoaz** [comment]
^ up

**Comment by dleslie** [comment]
Appears to still be broken when installing via elpa.

**Comment by dleslie** [comment]
@maxchaos Been a while, maybe time to merge?

**Comment by maxchaos** [comment]
@dleslie I guess so.

**Comment by fu123456** [comment]
Now, How to sove this issue to make this package work?

**Comment by marlinstrub** [comment]
I've used your fix for a while, @maxchaos, thanks for making it public! I noticed yesterday that it stopped working for me though. I'm just wondering whether it's still working for you and if not, whether you have any thoughts on how to fix it?

**Comment by maxchaos** [comment]
@marlinstrub Indeed, trying to fetch a page from powerthesaurus falls through with status code 403 (forbidden). It would appear that powerthesaurus is dropping http requests without certain header arguments (the reason why eludes me). Also, there was a change in the response format. I just pushed a fix on my fork's master branch that should work around both of these issues. Let me know if it works for you.

**Comment by marlinstrub** [comment]
@maxchaos Works like a charm, thanks a lot!

**Comment by marlinstrub** [comment]
Even better, thanks!
